org.eclipse.dawnsci.remotedataset.server/src/org/eclipse/dawnsci/remotedataset/server/info/TreeServlet.java

```diff
@@ -92,17 +96,28 @@ private void doHandle(HttpServletRequest  request,
 		
 		try {
 			final ILoaderService lservice = ServiceHolder.getLoaderService();
+			lservice.clearSoftReferenceCache();
+			long startTime = System.currentTimeMillis();
+			
 			final IDataHolder holder = lservice.getData(path, true, new IMonitor.Stub());
-					        
 			final Tree tree = holder.getTree();
+			
```

Comment:
This code is copied to the below file. It should be in the one method or breaks the DRY principle.